<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Scenario: Creating order with the same order id" tests="5" skipped="0" failures="0" errors="0" timestamp="2018-03-04T09:16:59" hostname="Keis-MBP" time="0.001">
  <properties/>
  <testcase name="Given There are several orders in order book" classname="Scenario: Creating order with the same order id" time="0.0"/>
  <testcase name="When Adding order with order id exists" classname="Scenario: Creating order with the same order id" time="0.0"/>
  <testcase name="Then It throws runtime exception" classname="Scenario: Creating order with the same order id" time="0.0"/>
  <testcase name="Then All orders in order book" classname="Scenario: Creating order with the same order id" time="0.0"/>
  <testcase name="classMethod" classname="Scenario: Creating order with the same order id" time="0.002"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
